junit5/src/main/java/sunstone/core/AzureUtils.java

```diff
@@ -0,0 +1,88 @@
+package sunstone.core;
+
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.identity.ClientSecretCredentialBuilder;
+import com.azure.resourcemanager.AzureResourceManager;
+import com.azure.resourcemanager.appservice.models.WebApp;
+import com.azure.resourcemanager.compute.models.VirtualMachine;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.wildfly.extras.sunstone.api.impl.ObjectProperties;
+import org.wildfly.extras.sunstone.api.impl.ObjectType;
+
+import java.io.IOException;
+import java.net.SocketTimeoutException;
+import java.util.Set;
+
+public class AzureUtils {
+    private static ObjectProperties objectProperties = new ObjectProperties(ObjectType.JUNIT5, null);
+
+    public static AzureResourceManager getResourceManager() {
+        return AzureResourceManager
```

Comment:
Just a suggestion. Since we don't expect the credentials to change, maybe we could cache the `AzureResourceManager` value for further calls? No idea how often you expect this to be used.

Reply:
This should not be private. This is not expected to be called outside of Sunstone. Yeah, it makes sense to cache Azure ARM client.

Reply:
Fixed.

Reply:
LGTM.